docs/dev-guides/index.md

```diff
@@ -7,23 +7,34 @@ We're thrilled to have you on board with making the Webaverse a more exciting pl
 
 The Webaverse runs on applications that you can write, package and distribute. These packages are called XRPackages (XRPKs).
 
-We've made it as easy as possible to get started! Please check out our [`XRPackage Overview`](./1-xrpackage-overview.md) to begin your developer journey into the Webaverse.
+We've made it as easy as possible to get started! A few points to start with:
 
-### Supported Asset Types
+- There's no Webaverse SDK! Just make your app using one of the many supported asset types (see below) and quickly package it using our CLI.
```

Comment:
We should mention doing packaging from the `xrpackage.org/inspect.html` page as well.

The inspect page does need work but doing things in drag and drop is something non-technical users tend to prefer for dealing with model assets.